lib/ui/components/mocked-requests.js

```diff
@@ -51,6 +51,39 @@ export class MockedRequests extends React.Component {
     API.toggleMockedRequest(mockedRequestId);
   }
 
+  _requestUrlFor(request) {
+    if (request.id === this.state.editingRequestId) {
+      return <input type="text"
+        autoFocus
+        value={ this.state.editedRequestUrl }
+        onChange={ this._handleRequestUrlChange.bind(this) }
+        onBlur={ this._submitRequestUrl.bind(this, request) }/>
+    } else {
+      return <div>
```

Comment:
So how does it work? Anyway, I don't even need the DIV, before I had 2 elements in there.
Will fix...